.github/pull_request_template.md

```diff
@@ -1,36 +1,15 @@
-<!--
-  Name your PR: (Use one of the below formats)
-  - [CLI-NNNN] Brief description of changes if jira ticket
-  - [WB-NNNN] Brief description of changes if jira ticket
-  - Brief description of changes
-
-  Also:
-  - Mark your PR as a Draft if it isnt ready for merge yet
--->
-
-<!-- Include one or more of the following issue URLs if applicable -->
-https://wandb.atlassian.net/browse/WB-NNNN
-https://wandb.atlassian.net/browse/CLI-NNNN
-https://github.com/wandb/client/issues/NNNN
+Fixes WB-NNNN
+Fixes #NNNN
 
 Description
 -----------
-
 What does the PR do?
 
 Testing
 -------
-
 How was this PR tested?
 
-Release Notes
--------------
-
-Below, please enter user-facing release notes as one or more bullet points.
-If your change is not user-visible, write `NO RELEASE NOTES` instead, with no bullet points.
-
-------------- BEGIN RELEASE NOTES ------------------
-
-
-
-------------- END RELEASE NOTES --------------------
+Checklist
+-------
+- Name PR "[WB-NNNN][WB-MMMM] Add support for..." similar to entries in CHANGELOG.md
+- Include reference to internal ticket "Fixes WB-NNNN" (and github issue "Fixes #NNNN" if applicable)
```

Comment:
```suggestion
[ ] Name PR "[WB-NNNN][WB-MMMM] Add support for..." similar to entries in `CHANGELOG.md`
[ ] Include reference to internal ticket "Fixes WB-NNNN" (and github issue "Fixes #NNNN" if applicable)
```

Reply:
i intentionally didnt use task lists giving a user a chance to use their own task list for their own items.   Im open for it though if you think it makes sense.

My work flow when i create a WIP PR is to make my own task list and then know im done when the tasks are complete.  Adding these fixed elements feel like a different class of tasks.   I think if (when) we have a more complete list of things it might make more sense to have a tasklist

Reply:
No strong feeling! Your reasoning makes sense :)